### ISSUE TITLE
Set a default pdf viewer

### ISSUE THREAD

**Issue by baltitenger** [ioi-2023/contestant-vm#81]
**Describe the bug**
Trying to open a pdf from firefox opens a popup to choose between document viewer and imagemagick.

**Expected behavior**
It should just open the pdf.

**Screenshots**
![image](https://github.com/ioi-2023/contestant-vm/assets/21960678/45ac6340-1bbd-477e-84c7-879b6e2a0f49)

**Please complete the following information:**

 - For pre-built, image release version: version-0804-cms
 - For manual build, commit: [none]
   - I have used autoinstall: [yes/no]

**Comment by radl97** [comment]
```bash
$ xdg-mime query default application/pdf
org.gnome.Evince.desktop
```

**Comment by radl97** [comment]
No, it's the same in the VM. Even `xdg-open` works without prompting.

**Comment by baltitenger** [comment]
I remember firefox doing its own thing instead of using xdg-mime... Not sure what needs to be set and where then.

**Comment by radl97** [comment]
Yes, this is that. Even from file chooser (nautilus) it seems to work with a good default.

**Comment by radl97** [comment]
I have investigated, and it is more of an annoyance than anything. I'll come ul with some way to tag issues like this that there are known workarounds, without apparent fixes.

**Comment by radl97** [comment]
Also this is not specifically a bug

**Comment by baltitenger** [comment]
Yea, the template added it. Not sure if there was a way to stop it.

**Comment by radl97** [comment]
Ah, I understand. Feel free to modify labels (if you have permission) or bypass the template entirely:

> Don’t see your issue here? [Open a blank issue.](about:blank)

**Comment by radl97** [comment]
`pdfjs.enabledCache.state` seems the relevant option. (to put to `user.js`)
